worker-batch-extensibility/src/main/java/com/hpe/caf/worker/batch/BatchWorkerPlugin.java

```diff
@@ -34,7 +34,11 @@
      * @param taskMessageType String containing the task message type as which to base the construction of the task data
      *                        on
      * @param taskMessageParams Map containing additional task message parameters
+     * 
+     * @throws BatchDefinitionException if the batch could not be processed
```

Comment:
It's not strictly related to this Jira but rather than "if the batch could not be processed" the description for when that exception should be thrown should be something like "if the batch definition is invalid".